assets/js/modules/materials.js

```diff
@@ -19,6 +19,7 @@ function init() {
     let storeLangPrefInForm = newState => $('#js-language-choice').val(newState);
 
     new Vue({
+        delimiters: ['<%', '%>'],
```

Comment:
I would prefer to set this globally if possible rather than on each module, otherwise it breaks pages in hilarious ways if we forget it (gets confused with Nunjucks' `{{ syntax }}`. It's a sitewide config rather than a per-instance one, you know?